src/AEOS/AEOSSyncTool/Adapters/SmartFaceDataAdapter.cs

```diff
@@ -0,0 +1,335 @@
+            }
+            if (SmartFaceDefaultThreshold <= 0)
+            {
+                throw new InvalidOperationException("The SmartFace threshold needs to be greater than 0.");
+            }
+            if (MaxFaces <= 0)
+            {
+                throw new InvalidOperationException("The MaxFace value should be at least 1.");
+            }
+            if (MaxFaceSize <= 0)
+            {
+                throw new InvalidOperationException("The MaxFaceSize needs to be a positive value.");
+            }
+            if (MinFaceSize <= 0)
+            {
+                throw new InvalidOperationException("The MinFaceSize needs to be a positive value.");
+            }
+            if (ConfidenceThreshold <= 0)
+            {
+                throw new InvalidOperationException("The ConfidenceThreshold needs to be a positive value.");
+            }
+            if (AeosWatchlistName == null)
+            {
+                throw new InvalidOperationException("The watchlist name for importing into SmartFace is necessary.");
+            }
+        }
+
+        public async Task<IList<SmartFaceMember>> getEmployees()
+        {
+            if (SmartFaceURL is null)
+            {
+                throw new ArgumentNullException(nameof(SmartFaceURL));
+            }
+
+            this.logger.Debug("Receiving Employees from SmartFace");
+
+            var SyncedWatchlists = new List<string>();
+
+            if (SmartFaceSyncedWatchlists.Count > 0)
+            {
+                this.logger.Debug($"SmartFaceSyncedWatchlists[]: {string.Join(" ", SmartFaceSyncedWatchlists.Select(i => i.Key))}");
+
+
+                foreach (var item in SmartFaceSyncedWatchlists)
+                {
+                    if (item.Value == true)
+                    {
+                        SyncedWatchlists.Add(new String(item.Key));
+                    }
+                }
+                this.logger.Information($"SyncedWatchlists[]: {string.Join(" ", SyncedWatchlists)}");
+            }
+            else
+            {
+                this.logger.Information("SmartFaceSyncedWatchlists is empty");
+            }
+
+            var SmartFaceAllMembers = new List<SmartFaceMember>();
+
+            if (SyncedWatchlists.Count() == 0)
+            {
+                bool allMembers = false;
+
+                while (allMembers == false)
+                {
+                    {
+                        var watchlistMembers = await graphQlClient.GetWatchlistMembers.ExecuteAsync(SmartFaceAllMembers.Count, SmartFaceSetPageSize);
+                        foreach (var wm in watchlistMembers.Data.WatchlistMembers.Items)
+                        {
+                            var imageDataId = wm.Tracklet.Faces.OrderBy(f => f.CreatedAt).FirstOrDefault(f => f.FaceType == AEOSSyncTool.FaceType.Regular)?.ImageDataId;
+                            this.logger.Information($"SF: {wm.Id}\t{imageDataId}\t{wm.DisplayName}");
```

Comment:
please do not add \t and such special/formatting characters to log output. Should be unformatted plain text.